product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/01_pem_probes.mdx

```diff
@@ -9,11 +9,17 @@ legacyRedirectsGenerated:
 
 A `probe` is a scheduled task that retrieves information about the database objects that are being monitored by the PEM agent. PEM uses the collected information to build the graphs displayed on each homepage. The `Manage Probes` tab (accessed via the `Management` menu) allows you to modify the data collection schedule and the length of time that PEM will retain information returned by a specific probe.
 
-Unless otherwise noted, Postgres Enterprise Manager enables the probes listed in the table below:
+Unless otherwise noted, Postgres Enterprise Manager enables the probes listed in the table below at server, database, schema, extension(from PEM 8.4 onwards), or agent levels:
```

Comment:
```suggestion
Unless otherwise noted, Postgres Enterprise Manager enables the following probes at the server, database, schema, extension (starting with version 8.4), or agent levels:
```

Reply:
Are there other levels? If this is a comprehensive list we might consider having a New Feature note about extensions. Might make sense to do that regardless. For the table, I would remove "This probe" from the start of each description and start with the verb instead.